components/Python/Scikit-learn/model_connector_cloudera_cdsw/main.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import print_function
```

Comment:
Why model connector is under Sckit-learn?

Reply:
Please note, Model connector on CDSW is independent of Scikit-learn, please have it under components/Python/model_connector_cloudera_cdsw

Reply:
A kind request please have @PMjoydas included in the review for model-connectors and predictor being added for CDSW/H2o